src/ophyd_async/plan_stubs/_fly.py

```diff
@@ -38,13 +38,12 @@ def prepare_static_pcomp_flyer_and_detectors(
 def prepare_static_seq_table_flyer_and_detectors_with_same_trigger(
     flyer: StandardFlyer[SeqTableInfo],
     detectors: list[StandardDetector],
-    number_of_frames: int,
+    number_of_frames: int | list[int],
```

Comment:
In this plan stub, we are making number_of_frames * repeats frames.
This means number_of_frames should be int